src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person: { name, sex, age, partnerName, isMarried } }) => {
```

Comment:
Agree with gitHub
```suggestion
export const Person = ({ person }) => {
 const { 
   name, 
   sex, 
   age, 
   partnerName, 
   isMarried
 } = person;
```
![image](https://user-images.githubusercontent.com/91411692/231396215-1584e6ba-07e1-4504-8745-f33258394f2b.png)